api/spec/v1.yaml

```diff
@@ -2024,6 +2038,22 @@ components:
             The number of addresses that have a nonzero balance of this token,
             as calculated from Transfer events.
           example: 123
+    
+    AccountStats:
+      type: object
+      required: [num_txns]
```

Comment:
Why not also the total_* fields?

Reply:
Oh this was in case we added the stats to consensus accounts but couldn't derive the sent/received amounts easily. Since we'll add those in a different PR, I'll make all 3 fields required for now.